formats/definitions.json

```diff
@@ -550,6 +550,67 @@
           }
         }
       }
+    },
+    "hmrc_manual_change_notes": {
+      "description": "A history of changes to HMRC manuals and the associated section. section_id is included and required making it different to manual_change_notes",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "section_id",
```

Comment:
I'm going to approach this in a separate PR, I'd prefer to minimise the risk with this larger set of changes.